prereise/gather/demanddata/transportation_electrification/generate_scaling_factors.py

```diff
@@ -155,19 +156,14 @@ def calculate_vmt_for_ua(census_ua, tht_ua):
             r"[, -.]", "", regex=True
         )
         common = set(tht_ua_format.index).intersection(set(census_ua_format.index))
-        vmt_for_ua[s] = (
-            pd.DataFrame(
-                {
-                    "Annual VMT": [
-                        365 * tht_ua_format.loc[i] * census_ua_format.loc[i]
-                        for i in common
-                    ]
-                },
-                index=list(common),
-            )
-            .rename(index=format2original)
-            .squeeze()
```

Comment:
Remove `squeeze` operation since it converted one element data frame into `float`. We need the index (urban are name to generate scaling factors)